tensorflow/tools/pip_package/BUILD

```diff
@@ -157,6 +157,7 @@ COMMON_PIP_DEPS = [
     "//tensorflow/python/distribute/coordinator:remote_eager_lib",
     "//tensorflow/python/distribute/coordinator:metric_utils",
     "//tensorflow:tensorflow_py",
+    "//tensorflow/tools/docs/tf_doctest_lib.py",
```

Comment:
It should be the target that provides this, not the python file

Reply:
Thanks for the review :)